backend/src/v5/services/filesManager.js

```diff
@@ -214,4 +216,25 @@ FilesManager.storeFileStream = async (teamspace, collection, id, dataStream, met
 	await insertRef(teamspace, collection, { ...meta, ...refInfo, _id: id, mimeType });
 };
 
+FilesManager.getMD5FileHash = async (teamspace, container, filename) => {
+	const refEntry = await getRefEntry(teamspace, `${container}.history.ref`, filename);
+
+	if (Object.keys(refEntry).includes('MD5Hash')) {
```

Comment:
I think you can probably abbreviate this bit with ES6 style syntax?

Also can we call it `md5Hash` just for consistency in naming convention?

```suggestion
const {size, md5Hash: hash }= await getRefEntry(teamspace, `${container}.history.ref`, filename);

if(hash) {
  return {size, hash};
}
```